### DISCUSSION TITLE
Micro lag when scrolling in Tauri macOS version (WKWebView) but not in Safari.

### DISCUSSION THREAD

**Discussion by HugoOlsson** [tauri-apps/tauri#8436] [Q&A]
Hi, I am working on a project that uses a relatively simple layout. The problem is that the application has a very frequent and noticeable micro lag in the macOS version when scrolling. I tried in Chrome (using localhost) and it was completely smooth, so I thought that it was a difference between the rendering engines. However, I have now also tried in Safari using localhost and it is completely smooth there as well. Since WKWebView uses the same engine as Safari, something else must cause this. 

I have just recently updated macOS to ensure that I have the latest WKWebView version (19617.1.17.11.9), so the problems should not be because of an outdated version. The Safari version I am using is: ”Version 17.2 (19617.1.17.11.9)”.

May there be a setting in WKWebView to make it handle things the same as Safari? And if yes, can I change that setting in Tauri?

May it be that WKWebView doesn't prioritize to use resources like Safari?

Is there anything else that can affect this? I have optimized the layout, so it should not be heavy to scroll.

**Comment by HugoOlsson** [comment]
It's kind of like the app is locked in 60 Hz. I use a 120 Hz display so Safari and Chrome renders in 120 Hz. Can there be such limitation in max frame rate for the app?

**Comment by xixioba** [comment]
I have a similar problem, the app is limited to 60FPS and in the browser I get 120FPS
Adding [info.plist](https://tauri.app/v1/guides/building/macos/#application-bundle-customization) has no effect
Below is my **Info.plist**
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
<key>CADisableMinimumFrameDurationOnPhone</key><true/>
<key>CADisplayLinkMinimumFramesPerSecond</key>
<integer>120</integer>
<key>CADisableMinimumFrameDuration</key><true/>
</dict>
</plist>
```

**Comment by metasmash** [comment]
Hello

I don't know if this is related, but have you tried passing a different browser agent to your web view?
ref: https://github.com/tauri-apps/tauri/discussions/11957

**Comment by KinanLak** [comment]
Hello, any updates on this ?
Has anyone managed to run a Tauri app at 120 FPS on a Mac?

**Comment by jabeztadesse** [comment]
Also on Linux ...

**Reply by FabianLars** [reply]
do you mean compared with the Epiphany browser? Otherwise it's really not the same as the rest of this topic (macos webview and safari use the same engine while on linux there are barely any browsers using webkit(gtk))
